src/screens/FeedScreen/Header.js

```diff
@@ -100,13 +112,37 @@ const _renderAnonimity = ({
   height,
   headerStyle,
   showAnonymousOption = false,
-  onHeaderOptionClicked = () => {},
-  hideThreeDot,
   version = 1,
   anonUserInfo = {},
-  isPostDetail
+  isPostDetail,
+  isFollow = false,
+  onPressFollUnFoll = () => {},
+  onDeletePost = () => {},
+  isShowDelete = false,
+  isSelf = false
 }) => {
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

New props are added to handle follow/unfollow and delete post functionalities. A ref `refSheet` is created for controlling the bottom sheet menu, and an array `dataSheet` is initialized with options for the menu. This is a good approach to manage the state and behavior of the bottom sheet menu.
```diff
+    version = 1,
+    anonUserInfo = {},
+    isPostDetail,
+    isFollow = false,
+    onPressFollUnFoll = () => {},
+    onDeletePost = () => {},
+    isShowDelete = false,
+    isSelf = false
+  }) => {
+    const navigation = useNavigation();
+    const refSheet = React.useRef();
+    const dataSheet = [
+      {
+        id: 1,
+        name: 'Share link',
+        icon: <ShareAndroidIcon />
+      }
+    ];
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->